source/develop/release-management/features/network/networkfilterparameters.html.md

```diff
@@ -0,0 +1,282 @@
+oVirt manages the network filter parameters and VDSM uses the libVirt to apply them to the VMs.
+
+In the scenario of [Bug 1366905][4] network filter parameters are required to restrict the allowed IP addresses of a VM.
+Network filter parameters can not only be used to define allowed or blocked IP address,
+but also for restricting port ranges on transport layer or allowing only trusted DHCP servers.
+
+## Benefit to oVirt
+
+This feature enables the usages of oVirt in scenarios, in which the user has to set parameters,
+without forcing the user to edit configuration files on the hosts manually.
+
+### User work-flows
+
+To use network filter parameters, the following workflow is expected:
+
+1. The user may create a new logical network or decide to use an already existing one.
+2. The user may create a new vNIC Profile or decide to apply an already existing one to the logical network.
+3. The user ensures, that the logical network is assigned to the hosts of the cluster.
+4. The user instantiates a (virtual) network interface (vNIC) of a virtual machine by picking the vNIC profile. 
+5. The user configures applied vNIC Profile to use a network filter, which depends on parameters.
+6. The user creates network filter parameters for this network interface of the virtual machine.
+
+If the predefined logical network `ovirtmgmt` and the associated predefined vNIC Profile `ovirtmgmt` is used,
+there is nothing to do in the first three steps.
+
+### User Experience
+
+![UI mockup](../../../../images/wiki/NetworkFilterParameterGUIMockup.png  "UI mockup")
+The NetworkInterfacePopupWidget has to be extended by a new section to edit the network filter parameters
+associated to a network interface of the virtual machine.
+The new section provides the ability to add, edit and delete network filter parameters.
+If many network interface parameters are associated to the network interface, the section could be scrolled.
+Unfortunately there is no generic way to deduce the network filter parameters accepted by a network filter, so this section can not decide which parameter names are allowed.
+
+## Entity Description
+
+The top-level network filter is [associated to the vNIC profile][2]. For this reason it appears likely
+that the parameters for this filters, e.g. allowed networking protocols in a network,
+are associated to the vNIC profile, too.
+But the use case described in the [bug report][4] gives the VM's IP addresses as an example for parameters.
+These parameters are associated of a concrete network interface of the VM's instance.
+So there is a need for parameters on two places.
+
+In a first step, the parameters are implemented for VM's network interfaces,
+because it addresses the given use case in the [bug report][4].
+Furthermore there is the possibility to set parameters on VM's network interfaces,
+temporary solves the vNIC profile scenarios, too.
+The drawback of this approach is that in this scenarios more configuration needed,
```

Comment:
The drawback of this approach is that in this scenarios more configuration needed,
+than it would be of the vNIC profiles could manage the filter parameters.

->
The drawback of this approach is that more configuration is needed than it would be if the vNIC profiles could manage the filter parameters.